lucidsonicdreams/main.py

```diff
@@ -82,14 +96,49 @@ def __init__(self,
     self.num_possible_classes = num_possible_classes 
     self.style_exists = False
     
+    # some stylegan models cannot be converted to pytorch (wikiart)
+    self.use_tf = style in ("wikiart",)
```

Comment:
I think a nice way to handle the cases where TF is compatible and not torch would be to:

1 - Define a variable `engine` in the `LucidSonicDream` object
2 - Create a compatibility dictionary (example : `compatibility_dict = {'tf' : ['model_1','model_2'], 'torch' : ['model2']}`)
3 - Raise an error if the model chosen when the `LucidSonicDream` object is created is not compatible with the engine (tf or torch)